synapse/tests/test_cortex.py

```diff
@@ -2928,7 +2928,7 @@ async def test_remote_storm(self):
             self.len(1, [mesg for mesg in mesgs if mesg[0] == 'init'])
             self.len(1, [mesg for mesg in mesgs if mesg[0] == 'fini'])
             # Lark sensitive test
-            self.stormIsInErr("No terminal defined for '|'", mesgs)
+            self.stormIsInErr("Unexpected token Token('VBAR', '|')", mesgs)
```

Comment:
It feels almost a hair clunky to be pushing through the terminal names when we have the terminalEnglishMap struct to help decode them?

Reply:
Yeah I'm going to update the error messages to use our own strings instead of what lark gives us.

Reply:
updated error message to remove terminal names